src/components/course/CourseRunSelector.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext, useState, useRef } from 'react';
+import moment from 'moment';
+
+import { CourseContext } from './CourseContextProvider';
+import { SET_COURSE_RUN } from './data/constants';
+
+import './styles/CourseRunSelector.scss';
+
+export default function CourseRunSelector() {
+  const { dispatch, state } = useContext(CourseContext);
+  const {
+    activeCourseRun,
+    course: {
+      courseRuns,
+    },
+  } = state;
+  const [editing, setEditing] = useState(false);
+  const selectRef = useRef(activeCourseRun.uuid);
+  const multipleRunsAvailable = courseRuns && courseRuns.length > 1;
+
+  const handleClick = () => {
+    const selectedCourseRun = courseRuns.find((courseRun) => courseRun.uuid === selectRef.current.value);
+    dispatch({ type: SET_COURSE_RUN, payload: selectedCourseRun });
+    setEditing(false);
+  };
+
+  if (multipleRunsAvailable) {
+    if (!editing) {
+      return (
+        <button type="button" onClick={() => setEditing(true)} className="btn btn-link my-0 p-0">
+          <span className="link">
```

Comment:
nit: is this span with `className=link` necessary for styling?

Reply:
It is for prospectus and not for us, just something I didn't trim out, fixed!